fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -598,6 +599,11 @@ bool isCipherKeyEligibleForRefresh(const EncryptBaseCipherKey& cipherKey, int64_
 	return nextRefreshCycleTS > cipherKey.expireAt || nextRefreshCycleTS > cipherKey.refreshAt;
 }
 
+bool isBlobMetadataEligibleForRefresh(const BlobMetadataDetailsRef& blobMetadata, int64_t currTS) {
+	int64_t nextRefreshCycleTS = currTS + SERVER_KNOBS->BLOB_METADATA_REFRESH_INTERVAL;
```

Comment:
may want to BUGGIFY the code?

Reply:
Sure, that's reasonable. We should do the same in the encryption key code just above too

Reply:
Good point! We should do that, will take an AI for it.

Reply:
I'm just doing it as part of this change